src/trades/TradesTabs.js

```diff
@@ -28,6 +28,11 @@ export default class TradesTabs extends Component {
                         <Tab
                             key={index}
                             text={trade.symbolName}
+                            showCloseIcon
+                            onClose={ev => {
+                                actions.removeTrade(index);
+                                ev.stopPropagation();
```

Comment:
Used to adding those in js buttons to address event bubbling issue [https://developer.mozilla.org/en/docs/Web/API/Event/stopPropagation] here. Didn't try without it in react though . :wink: